### ISSUE TITLE
remember playback position

### ISSUE THREAD

**Issue by ee7klt** [quodlibet/quodlibet#3415]
in itunes, there's an option where i can resume playback from where i left off for all tracks. is there such a feature for quod libet?

**Comment by frestr** [comment]
Not exactly, but you can set bookmarks in a track using the "Go to Bookmark" plugin and right-clicking the time display. This has to be done manually though.

**Comment by declension** [comment]
I guess defining "left off" is part of the hard bit here too

See also #1331
